changelog.d/pa-2507.added

```diff
@@ -0,0 +1,2 @@
+Pro: Taint: Taint labels now work interprocedurally, for everything except
+labeled propagators.
```

Comment:
What happens then?

Reply:
They should work as before, but just not interprocedurally. You should not expect a labelled propagator to propagate if it is within a function call, because it should almost never propagate.